src/scripts.js

```diff
@@ -26,3 +29,8 @@ function displayFriends() {
   friends[2].innerText = arrayOfFriends[2];
   friends[3].innerText = arrayOfFriends[3];
 };
+
```

Comment:
Awesome work!  Let's see if we can't get the chart.js to display some sort of cool chart to show this; potentially like the fitbit app in our design inspirations wireframe.